libralgo/libral.go

```diff
@@ -1,4 +1,4 @@
-package libral
+package libralgo
```

Comment:
I assume this change needs to be merged into the previous to make sure builds would work, right ?